public/docs/features.md

```diff
@@ -11,121 +10,128 @@ If you experience any _issues_, feel free to report it on [**GitHub**](https://g
 Or meet us on [**Matrix.org**](https://riot.im/app/#/room/#codimd:matrix.org) for dev-talk and interactive help.
 **Thank you very much!**
 
-Workspace
-===
-## Modes
-**Desktop & Tablet**
+## Workspace
+
+### Modes
+
+#### Desktop & Tablet
 
 <i class="fa fa-edit fa-fw"></i> Edit: See only the editor.
 <i class="fa fa-eye fa-fw"></i> View: See only the result.
 <i class="fa fa-columns fa-fw"></i> Both: See both in split view.
 
-**Mobile**
+#### Mobile
 
 <i class="fa fa-toggle-on fa-fw"></i> View: See only the result.
 <i class="fa fa-toggle-off fa-fw"></i> Edit: See only the editor.
 
-## Night Mode:
+### Night Mode
+
 When you are tired of a white screen and like a night mode, click on the little moon <i class="fa fa-moon-o"></i> and turn on the night view of CodiMD.
 
 The editor view, which is in night mode by default, can also be toggled between night and day view using the the little sun<i class="fa fa-sun-o fa-fw"></i>.
 
-## Image Upload:
+### Image Upload
+
 You can upload an image simply by clicking on the camera button <i class="fa fa-camera"></i>.
```

Comment:
that camera button no longer exists, It is now 
![upload button editor](https://user-images.githubusercontent.com/124909/55509979-88e27300-565d-11e9-8e87-c09213e19fcb.png)

I'm not sure if this is included in fontawesome, but @SISheogorath may know?

Reply:
should be `fa-upload` instead of `fa-camera`

Reply:
whoops, I always mix those up.